lib/Zonemaster/Backend/RPCAPI.pm

```diff
@@ -431,10 +539,63 @@ sub add_batch_job {
     return $results;
 }
 
-
+$json_schemas{get_batch_job_result} = joi->object->strict->props(
+    batch_id => $Zonemaster::Backend::Validator::batch_id->required
+);
 sub get_batch_job_result {
     my ( $self, $batch_id ) = @_;
 
     return $self->{db}->get_batch_job_result($batch_id);
 }
+
+my $rpc_request = joi->object->props(
+    jsonrpc => joi->string->required,
+    id => joi->integer->min(0)->required,
+    method => joi->string->regex("[a-zA-Z0-9_-]*")->required
+);
+sub json_validate {
+    my ( $self, $json_schema) = @_;
```

Comment:
The argument here a parsed JSONRPC request, right? Not a schema. Could you rename the argument to be more helpful?

Reply:
Done in commit [5237906](https://github.com/dotse/zonemaster-backend/pull/399/commits/523790622e8243b5130dfd33be1c41f75a7215a7)